cmd/corald/midware/auth.go

```diff
@@ -1,136 +0,0 @@
-package midware
```

Comment:
Anvil was removed from vendored deps a while back actually. Refer to https://github.com/coralproject/shelf/blob/internal-auth-middleware/vendor/vendor.json and grep for `anvil`.